[2025-03-16T20:04:02.832+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: frequent_flyers.create_table scheduled__2025-03-15T00:00:00+00:00 [queued]>
[2025-03-16T20:04:02.837+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: frequent_flyers.create_table scheduled__2025-03-15T00:00:00+00:00 [queued]>
[2025-03-16T20:04:02.837+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-03-16T20:04:02.844+0000] {taskinstance.py:1380} INFO - Executing <Task(PostgresOperator): create_table> on 2025-03-15 00:00:00+00:00
[2025-03-16T20:04:02.847+0000] {standard_task_runner.py:57} INFO - Started process 211 to run task
[2025-03-16T20:04:02.849+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'frequent_flyers', 'create_table', 'scheduled__2025-03-15T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/frequent_flyers.py', '--cfg-path', '/tmp/tmpg1jrplnn']
[2025-03-16T20:04:02.850+0000] {standard_task_runner.py:85} INFO - Job 7: Subtask create_table
[2025-03-16T20:04:02.860+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-16T20:04:02.884+0000] {task_command.py:415} INFO - Running <TaskInstance: frequent_flyers.create_table scheduled__2025-03-15T00:00:00+00:00 [running]> on host 16ae585d7ab0
[2025-03-16T20:04:02.937+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='frequent_flyers' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-15T00:00:00+00:00'
[2025-03-16T20:04:02.938+0000] {sql.py:274} INFO - Executing: 
    CREATE SCHEMA IF NOT EXISTS presentation;
    
    CREATE TABLE IF NOT EXISTS presentation.frequent_flyers (
        created_at TIMESTAMP DEFAULT NOW(),
        passenger_id VARCHAR(20),
        passenger_name TEXT,
        flights_number INTEGER,
        purchase_sum NUMERIC(10, 2),
        home_airport CHAR(3),
        customer_group VARCHAR(5),
        CONSTRAINT pk_frequent_flyers PRIMARY KEY (passenger_id)
    );
[2025-03-16T20:04:02.944+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-16T20:04:02.951+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-16T20:04:02.955+0000] {sql.py:418} INFO - Running statement: 
    CREATE SCHEMA IF NOT EXISTS presentation;
    
    CREATE TABLE IF NOT EXISTS presentation.frequent_flyers (
        created_at TIMESTAMP DEFAULT NOW(),
        passenger_id VARCHAR(20),
        passenger_name TEXT,
        flights_number INTEGER,
        purchase_sum NUMERIC(10, 2),
        home_airport CHAR(3),
        customer_group VARCHAR(5),
        CONSTRAINT pk_frequent_flyers PRIMARY KEY (passenger_id)
    );, parameters: None
[2025-03-16T20:04:02.973+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=frequent_flyers, task_id=create_table, execution_date=20250315T000000, start_date=20250316T200402, end_date=20250316T200402
[2025-03-16T20:04:02.989+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-16T20:04:03.005+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-16T20:12:25.855+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: frequent_flyers.create_table scheduled__2025-03-15T00:00:00+00:00 [queued]>
[2025-03-16T20:12:25.863+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: frequent_flyers.create_table scheduled__2025-03-15T00:00:00+00:00 [queued]>
[2025-03-16T20:12:25.863+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-03-16T20:12:25.871+0000] {taskinstance.py:1380} INFO - Executing <Task(PostgresOperator): create_table> on 2025-03-15 00:00:00+00:00
[2025-03-16T20:12:25.873+0000] {standard_task_runner.py:57} INFO - Started process 201 to run task
[2025-03-16T20:12:25.875+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'frequent_flyers', 'create_table', 'scheduled__2025-03-15T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/frequent_flyers.py', '--cfg-path', '/tmp/tmpj2n_ptma']
[2025-03-16T20:12:25.877+0000] {standard_task_runner.py:85} INFO - Job 7: Subtask create_table
[2025-03-16T20:12:25.892+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-16T20:12:25.917+0000] {task_command.py:415} INFO - Running <TaskInstance: frequent_flyers.create_table scheduled__2025-03-15T00:00:00+00:00 [running]> on host b70acf7f0e92
[2025-03-16T20:12:25.967+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='frequent_flyers' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-15T00:00:00+00:00'
[2025-03-16T20:12:25.968+0000] {sql.py:274} INFO - Executing: 
CREATE TABLE IF NOT EXISTS presentation.frequent_flyers (
    created_at TIMESTAMP DEFAULT NOW(),
    passenger_id VARCHAR(20),
    passenger_name TEXT,
    flights_number INTEGER,
    purchase_sum NUMERIC(10, 2),
    home_airport CHAR(3),
    customer_group VARCHAR(5),
    CONSTRAINT pk_frequent_flyers PRIMARY KEY (passenger_id)
);
[2025-03-16T20:12:25.975+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-16T20:12:25.981+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-16T20:12:25.986+0000] {sql.py:418} INFO - Running statement: 
CREATE TABLE IF NOT EXISTS presentation.frequent_flyers (
    created_at TIMESTAMP DEFAULT NOW(),
    passenger_id VARCHAR(20),
    passenger_name TEXT,
    flights_number INTEGER,
    purchase_sum NUMERIC(10, 2),
    home_airport CHAR(3),
    customer_group VARCHAR(5),
    CONSTRAINT pk_frequent_flyers PRIMARY KEY (passenger_id)
);, parameters: None
[2025-03-16T20:12:26.005+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=frequent_flyers, task_id=create_table, execution_date=20250315T000000, start_date=20250316T201225, end_date=20250316T201226
[2025-03-16T20:12:26.015+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-16T20:12:26.033+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-16T20:25:14.006+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: frequent_flyers.create_table scheduled__2025-03-15T00:00:00+00:00 [queued]>
[2025-03-16T20:25:14.012+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: frequent_flyers.create_table scheduled__2025-03-15T00:00:00+00:00 [queued]>
[2025-03-16T20:25:14.012+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-03-16T20:25:14.019+0000] {taskinstance.py:1380} INFO - Executing <Task(PostgresOperator): create_table> on 2025-03-15 00:00:00+00:00
[2025-03-16T20:25:14.022+0000] {standard_task_runner.py:57} INFO - Started process 233 to run task
[2025-03-16T20:25:14.024+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'frequent_flyers', 'create_table', 'scheduled__2025-03-15T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/frequent_flyers.py', '--cfg-path', '/tmp/tmpvf8lfk_z']
[2025-03-16T20:25:14.025+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask create_table
[2025-03-16T20:25:14.036+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-16T20:25:14.061+0000] {task_command.py:415} INFO - Running <TaskInstance: frequent_flyers.create_table scheduled__2025-03-15T00:00:00+00:00 [running]> on host 711ad4f92c48
[2025-03-16T20:25:14.112+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='frequent_flyers' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-15T00:00:00+00:00'
[2025-03-16T20:25:14.113+0000] {sql.py:274} INFO - Executing: 
CREATE TABLE IF NOT EXISTS presentation.frequent_flyers (
    created_at TIMESTAMP DEFAULT NOW(),
    passenger_id VARCHAR(20),
    passenger_name TEXT,
    flights_number INTEGER,
    purchase_sum NUMERIC(10, 2),
    home_airport CHAR(3),
    customer_group VARCHAR(5),
    CONSTRAINT pk_frequent_flyers PRIMARY KEY (passenger_id)
);
[2025-03-16T20:25:14.120+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-16T20:25:14.127+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-16T20:25:14.132+0000] {sql.py:418} INFO - Running statement: 
CREATE TABLE IF NOT EXISTS presentation.frequent_flyers (
    created_at TIMESTAMP DEFAULT NOW(),
    passenger_id VARCHAR(20),
    passenger_name TEXT,
    flights_number INTEGER,
    purchase_sum NUMERIC(10, 2),
    home_airport CHAR(3),
    customer_group VARCHAR(5),
    CONSTRAINT pk_frequent_flyers PRIMARY KEY (passenger_id)
);, parameters: None
[2025-03-16T20:25:14.148+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=frequent_flyers, task_id=create_table, execution_date=20250315T000000, start_date=20250316T202514, end_date=20250316T202514
[2025-03-16T20:25:14.165+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-16T20:25:14.182+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
